config/settings/base.py

```diff
@@ -609,6 +609,20 @@
     },
 }
 
+# Time estimates for reviewing tasks based on the assigned story points
+#
+# SPRINT_HOURS_RESERVED_FOR_REVIEW {
```

Comment:
```suggestion
{
```

The variable name looks redundant here.